docs/how_to_add_a_notifier.md

```diff
@@ -27,15 +27,11 @@ Finally, you need to modify the `lambda_py/notifiers/__init__.py` file:
 
 You will need to modify the `variables.tf` file:
 
-1. Add a variable specific to your notifier, for example `Custom_notifier`. This variable must have two fields:
+1. Add a variable specific to your notifier, for example `Custom_notifier`. This following field is mandatory:
 
-- `enabled` (`bool`), which will specify whether this notifier is enabled
-- `parameters` (`object`), which will hold configuration bits specific to this notifier. The subfields of the `parameters` field will be passed as-is to the lambda function, as environment variables. This means `parameters` should be a mapping from string to string.
+- `py_params` (`object`), which will be used to instantiate the associated python class (i.e. contains the field expected by the `__init__` method).
```

Comment:
Naming is strange since we don't need to use Python for it to work. why do we need two levels of params ?

Reply:
The thing is that we initialise notifiers using `**notifier["parameters"]`. However the SES notifier has parameter that is not passed to python (`zone_id`). So we chose this in order to pass exactly the needed parameters to the notifiers.

Reply:
In order to have all the parameters at the same level, we could select the parameters to pass based on the signature of the notifier.

Reply:
Yes, or to be simpler, we could allow extra arguments in the instanciation method of the notifier.

Reply:
Allowed extra args in notifiers __init__, and put all configuration at the same level.